ElrondNetwork/elrond-go
# EN-13278: move webserver antiflood configuration

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- The `Antiflood` enabled config value was not taken into consideration for the webserver section
  
## Proposed Changes
- Move the webserver antiflood values to a new configuration structure
- Don't use any limitation if the web server antiflood is disabled

## Testing procedure
1) general throttlers
- set config.toml `WebServerAntifloodEnabled` to `true`, `SameSourceRequests` to `2` and `SameSourceResetIntervalInSec` to `10` and query the endpoint `<node-ip>:<node:port"/network/config` few times. you should receive a "too many requests" response
- for the above scenario, set `WebServerAntifloodEnabled` to `false` and try the queries again. This time, the "too many requests" message should not be returned

2) trie operations
- set config.toml `WebServerAntifloodEnabled` to `true`, and`TrieOperationsDeadlineMilliseconds` to `1`. Query the endpoint `<node-ip>:<node:port"/network/esdts` on a testnet where there are a lot of tokens. It should fail with trie timeout
- for the above scenario, set  `WebServerAntifloodEnabled` to `false` and try the query again. This time, it should work
